README.md

```diff
@@ -48,12 +48,12 @@ To run this pipeline you will need the following inputs:
 * Input sample sheet
 
 You will need to create a sample sheet with information about the samples you are processing, before running the pipeline. 
-This file must be **tab-separated** and contain a header and one row per sample. Columns should correspond to sampleID, read1, read2 files:
+This file must be **tab-separated** and contain a header and one row per sample. Columns should correspond to sampleID and read location:
```

Comment:
This section needs to explain more clearly how paired-end fastq or samples with multiple fastq are treated. Based on the current description, it says one row per sample, yet based on the actual code in fastqc.nf, its evident that the input tsv file requires one row per FASTQ, and that samples with more than one fastq (be it pairs, or multiple singles) need to span multiple lines. While this works, the format of this tsv will not be directly compatible with other downstream processing steps such as alignment, so this will harm the modularity of the workflow.

Reply:
Have changed to csv, which should be standard for all our pipelines. Its less error prone.